### PR TITLE
Improvements to CONTRIBUTING.ejs

### PR DESCRIPTION
**Pull request by silviubogan**

### CODE REVIEW

File: lib/templates/CONTRIBUTING.ejs [hunk 1]
```diff
@@ -40,11 +42,11 @@ requested. :sparkles:
 ## Creating a pull request
 
  - Open a pull request, and reference the initial issue in the pull request
-   message (e.g. *fixes #<your-issue-number*). Write a good description and
+   message (e.g. *fixes #<your-issue-number>*). Write a good description and
```
Comment (IonicaBizau) [inline lib/templates/CONTRIBUTING.ejs]: Can you please test if `blah -c` renders this correctly? I hope it will not consider it as ejs tag. :balloon:

**Comment by IonicaBizau** [comment]
...and add yourself as contributor. :sparkle:

**Comment by silviubogan** [comment]
Was the removal of the comma [here](https://github.com/IonicaBizau/node-blah/commit/e97ecfa8bd191128893ea2762e4a1cd7201d19b8#diff-ab7ae2aa3db999795895cd5fc1e595fcL9) intentional?

File: lib/templates/CONTRIBUTING.ejs (hunk 1 above)
Comment (silviubogan) [inline lib/templates/CONTRIBUTING.ejs]: I tested `blah -c` and it works well, it does not consider it an EJS tag. :smiley:

**Comment by IonicaBizau** [comment]
@silviubogan Probably yes, but if it's better with comma add it back. :smile:

**Comment by silviubogan** [comment]
@IonicaBizau, I think everything is well done now. :smiley:

**Comment by IonicaBizau** [comment]
Thanks! :sparkles:

**Pull request merged**